megamek/src/megamek/common/verifier/TestAero.java

```diff
@@ -430,72 +388,24 @@ public static float calculateMaxTurnsAtSafe(Aero aero){
      * @param aero
      * @return
      */
-    public static float calculateMaxTurnsAtMax(Aero aero){
+    public static float calculateMaxTurnsAtMax(Aero aero) {
         int fuelPoints = aero.getFuel();
         float fuelPerTurn;
         if (aero.hasETypeFlag(Entity.ETYPE_CONV_FIGHTER)) {
             fuelPerTurn = aero.getWalkMP() * 0.5f;
-            if(aero.hasEngine()) {
-                if(aero.getEngine().isFusion()) {
-                    fuelPerTurn += (aero.getRunMP()-aero.getWalkMP()) * 2;
+            if (aero.hasEngine()) {
+                if (aero.getEngine().isFusion()) {
+                    fuelPerTurn += (aero.getRunMP() - aero.getWalkMP()) * 2;
                 } else {
-                    fuelPerTurn += (aero.getRunMP()-aero.getWalkMP());
+                    fuelPerTurn += (aero.getRunMP() - aero.getWalkMP());
                 }
             }
         } else if (aero.getWalkMP() == 0) {
             fuelPerTurn = 0.2f;
         } else {
-            fuelPerTurn = aero.getWalkMP() + 
-                    (aero.getRunMP()-aero.getWalkMP()) * 2;
-        }
-        return fuelPoints/fuelPerTurn;       
-    }    
-
-    public static int weightFreeHeatSinks(Aero aero) {
-        if (aero.hasETypeFlag(Entity.ETYPE_SMALL_CRAFT)) {
```

Comment:
Used by HeatSinkView in MML.
I did not check this for all removed methods, only for Engine.getTechType which does not seem to be used in MML.

Reply:
I'm too used to building everything, not working solely on MegaMek... restored them and checked MekHQ for uses too.